src/send/Send.tsx

```diff
@@ -126,22 +128,27 @@ function Send({ route }: Props) {
     // interfere with the invite modal or bottom sheet.
     Keyboard.dismiss()
 
-    if (
-      recipientVerificationStatus === RecipientVerificationStatus.UNVERIFIED ||
-      !hasAddressField(recipient) // should already be caught by UNVERIFIED and UNKNOWN checks, but doing this to be safe and to satisfy the TS compiler
-    ) {
+    if (recipientVerificationStatus === RecipientVerificationStatus.UNVERIFIED) {
       setShowInviteModal(true)
       return
     }
 
     if (getFeatureGate(StatsigFeatureGates.USE_NEW_SEND_FLOW)) {
-      navigate(Screens.SendEnterAmount, {
-        isFromScan: false,
-        defaultTokenIdOverride,
-        forceTokenId,
-        recipient,
-        origin: SendOrigin.AppSendFlow,
-      })
+      if (hasAddressField(recipient)) {
```

Comment:
nit: looks like we have duplicated helpers for this
https://github.com/valora-inc/wallet/blob/7ff040fa585fa3beb70e2a2b2ee0abe9d1558ad3/src/recipients/recipient.ts#L52-L54


```suggestion
      if (recipientHasAddress(recipient)) {
```